### PR TITLE
USB: Single page for events

### PR DESCRIPTION
**Pull request by Elchi3**
Use a single page for USB events.

BCD work: https://github.com/mdn/browser-compat-data/pull/13886

### CODE REVIEW

File: files/en-us/web/api/usb/disconnect_event/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+```js
+addEventListener('disconnect', event => { });
+
+ondisconnect = event => { };
+```
+
+## Event type
+
+An {{domxref("USBConnectionEvent")}}. Inherits from {{domxref("Event")}}.
```
Comment (dominiccooney) [inline files/en-us/web/api/usb/disconnect_event/index.md:28]: Nit: Article should be "*A* USBConnectionEvent

File: files/en-us/web/api/usb/index.md [hunk 2]
```diff
@@ -18,20 +18,21 @@ The **`USB`** interface of the [WebUSB API](/en-US/docs/Web/API/WebUSB_API) prov
...
 
-### Event handlers
-
-- {{domxref("USB.onconnect")}}
-  - : An event handler called whenever a previously paired device is connected.
-- {{domxref("USB.ondisconnect")}}
-  - : An event handler called whenever a paired device is disconnected.
-
-## Methods
+# Methods
```
Comment (dominiccooney) [inline files/en-us/web/api/usb/index.md:22]: Not related to your change, but should this page mention that you use `navigator.usb` to get one of these objects?

File: files/en-us/web/api/usb/connect_event/index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  - API
+  - Event
+  - Reference
+  - USB
+  - WebUSB
+browser-compat: api.USB.connect_event
+---
+{{APIRef("WebUSB API")}}{{SeeCompatTable}}
+
+The **`connect`** property of the {{DOMxRef("USB")}} interface is an event fired whenever a paired device is connected.
```
Comment (jpmedley) [inline files/en-us/web/api/usb/connect_event/index.md:14]:
```suggestion
The **`connect`** event callback of the {{DOMxRef("USB")}} interface is fired whenever a paired device is connected.
```

Comment (jpmedley) [reply files/en-us/web/api/usb/connect_event/index.md:14]: I think this conveys more information in roughly the same space. I can make this a formal proposal if need be.

File: files/en-us/web/api/usb/disconnect_event/index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  - API
+  - Event
+  - Reference
+  - USB
+  - WebUSB
+browser-compat: api.USB.disconnect_event
+---
+{{APIRef("WebUSB API")}}{{SeeCompatTable}}
+
+The **`connect`** property of the {{DOMxRef("USB")}} interface is an event fired whenever a paired device is disconnected.
```
Comment (jpmedley) [inline files/en-us/web/api/usb/disconnect_event/index.md:14]:
```suggestion
The **`connect`** event callback of the {{DOMxRef("USB")}} interface is fired whenever a paired device is disconnected.
```

File: files/en-us/web/api/usb/index.md [hunk 5]
```diff
@@ -32,6 +31,14 @@ None.
 - {{domxref("USB.requestDevice()")}}
   - : Returns a {{jsxref("Promise")}} that resolves with an instance of {{domxref("USBDevice")}} if the specified device is found. Calling this function triggers the user agent's pairing flow.
 
+## Events
+
+- {{domxref("USB.connect_event", "connect")}}
+  - : An event fired whenever a previously paired device is connected.
```
Comment (jpmedley) [inline files/en-us/web/api/usb/index.md:37]:
```suggestion
  - : Fired whenever a previously paired device is connected.
```

File: files/en-us/web/api/usb/index.md [hunk 6]
```diff
@@ -32,6 +31,14 @@ None.
...
+- {{domxref("USB.disconnect_event", "disconnect")}}
+  - : An event fired whenever a paired device is disconnected.
```
Comment (jpmedley) [inline files/en-us/web/api/usb/index.md:39]:
```suggestion
  - : Fired whenever a paired device is disconnected.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Pull request merged**